### PR TITLE
chore: rename review table

### PR DESCRIPTION
**Pull request by BolajiOlajide**
#### What Does This PR Do?
Rename the Review table to `Ratings`

### CODE REVIEW

File: reactlibapp/libraryapp/models.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from django.db import models
 from django.contrib.auth.models import User
+from likert_field.models import LikertField
```
Comment (andela-sjames) [inline reactlibapp/libraryapp/models.py]: Will this field work with Django Rest FrameWork?

Comment (andela-sjames) [reply reactlibapp/libraryapp/models.py]: @BolajiOlajide What is the state of this task?

Comment (BolajiOlajide) [reply reactlibapp/libraryapp/models.py]: It's done. The model field has been reverted to contain an `integer field`.

**Comment by andela-sjames** [comment]
@BolajiOlajide have you rebased with branch `develop` now?

**Comment by BolajiOlajide** [comment]
Yes @andela-sjames

**Pull request merged**